Assignments/Splitwise/CPP/main.cpp

```diff
@@ -7,9 +7,9 @@ class Users
     void addUsers(vector<string>);
     void getUser(string);
     void getUser();
-    void addExpense(string, string, string, vector<string>);
-    void addExpense(string, vector<string>, vector<string>);
-    void addExpense(string, vector<string>, vector<string>, string);
+    void addExpense(string, string, string, vector<string>);  //for EQUAL
```

Comment:
Well, this is still not what I'm expecting. Think of how you would use this app and how would you design. How you as a developer add incoming features in future versions. Would you end up overloading 100's of Add expense methods

Reply:
Removed multiple addExpense function. And split the code in multiple files.

Reply:
No that is not a viable solution either